xclim/sdba/adjustment.py

```diff
@@ -215,6 +217,29 @@ class DetrendedQuantileMapping(EmpiricalQuantileMapping):
     .. [Cannon2015] Cannon, A. J., Sobie, S. R., & Murdock, T. Q. (2015). Bias correction of GCM precipitation by quantile mapping: How well do methods preserve changes in quantiles and extremes? Journal of Climate, 28(17), 6938–6959. https://doi.org/10.1175/JCLI-D-14-00754.1
     """
 
+    @parse_group
+    def __init__(
```

Comment:
This class is missing a Parameters section in the docstring.

Reply:
True. It didn't need one before, but with `detrending` it now does.